docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-cloud-providers/amazon.md

```diff
@@ -159,6 +163,309 @@ Setting the value of the tag to `owned` tells the cluster that all resources wit
 
 **Key** = `kubernetes.io/cluster/CLUSTERID` **Value** = `shared`.
 
+:::note
+
+Do not tag a resource with multiple owned or shared tags.
+
+:::
+
 ### Using Amazon Elastic Container Registry (ECR)
 
 The kubelet component has the ability to automatically obtain ECR credentials, when the IAM profile mentioned in [Create an IAM Role and attach to the instances](#1-create-an-iam-role-and-attach-to-the-instances) is attached to the instance(s). When using a Kubernetes version older than v1.15.0, the Amazon cloud provider needs be configured in the cluster. Starting with Kubernetes version v1.15.0, the kubelet can obtain ECR credentials without having the Amazon cloud provider configured in the cluster.
+
+### Using the Out-of-Tree AWS Cloud Provider for RKE1
+
+1. [Node name conventions and other prerequisities ](https://cloud-provider-aws.sigs.k8s.io/prerequisites/) must be followed so that the cloud provider can find the instance. Rancher provisioned clusters don't support configuring `providerID`.
+
+> When IP-based naming is used, the nodes must be named after the instance followed by the regional domain name (`ip-xxx-xxx-xxx-xxx.ec2.<region>.internal`). If you have a custom domain name set in the DHCP options, you must set `--hostname-override` on `kube-proxy` and `kubelet` to match this naming convention.
+
+When creating a [custom cluster](../../../../pages-for-subheaders/use-existing-nodes.md), you must add [`--node-name`](../../reference-guides/cluster-configuration/rancher-server-configuration/use-existing-nodes/rancher-agent-options) to the `docker run` node registration command to set `hostname-override` -- for example, `"$(hostname -f)"`). This can be done manually or by using **Show Advanced Options** in the Rancher UI to add **Node Name**.
```

Comment:
@jakefhyde This line is still a problem. I returned to the Rancher UI hoping I would find something like these instructions but I can't confirm where users should go.

> This can be done manually or by using **Show Advanced Options** in the Rancher UI to add **Node Name**.

Reply:
![image](https://github.com/rancher/rancher-docs/assets/33796120/d1cc56c7-6a74-4fe3-ae70-5d5cd33920b1)
![image](https://github.com/rancher/rancher-docs/assets/33796120/cac8ad6c-0822-4df0-bcb2-6fd713c73a11)